6-AdvancedScenarios/1-call-api-obo/README.md

```diff
@@ -1,427 +1,361 @@
-# On-behalf-of flow and Conditional Access challenges
-
- 1. [Overview](#overview)
- 1. [Scenario](#scenario)
- 1. [Contents](#contents)
- 1. [Prerequisites](#prerequisites)
- 1. [Setup](#setup)
- 1. [Registration](#registration)
- 1. [Running the sample](#running-the-sample)
- 1. [Explore the sample](#explore-the-sample)
- 1. [About the code](#about-the-code)
- 1. [More information](#more-information)
- 1. [Community Help and Support](#community-help-and-support)
- 1. [Contributing](#contributing)
+---
+page_type: sample
+name: Handling Conditional Access challenges in an Azure AD protected Node.js web API calling another protected Node.js web API on behalf of a user
+description: Handling Conditional Access challenges in an Azure AD protected Node.js web API calling another protected Node.js web API on behalf of a user
```

Comment:
>Conditional Access challenges [](http://example.com/codeflow?start=22&length=29)

CA or cAE or both?

Reply:
Not supporting CA and CAE due to issue with MSAL. I updated to `Authenticate a user with Azure AD using msal.js and call an Azure AD protected Node.js Web Api using on-behalf of flow`.